Hangfire.Redis.StackExchange/project.json

```diff
@@ -1,23 +1,14 @@
 ﻿{
-  "dependencies": {
-    "NETStandard.Library": "1.6.0",
-    "Hangfire.Core": "1.6.5",
-    "StackExchange.Redis": "1.1.608"
-  },
+    "dependencies": {
```

Comment:
Tip: Best to match existing indentation, giving less visual changes in diffs (and hence PR)